setup.py

```diff
@@ -93,17 +93,15 @@ def maybe_append_with_mkl(dependencies):
         torch_config = torch.__config__.show()
         with_mkl_blas = 'BLAS_INFO=mkl' in torch_config
         if torch.backends.mkl.is_available() and with_mkl_blas:
-            # product version is decoupled from library version. For older
-            # releases, where MKL was not a part of oneAPI, we can safely use
-            # 2021.4 as it is backward compatible (default for PyTorch conda
-            # distribution).
-            product_version = '2021.4'
+            product_version = '2023.1.0'
```

Comment:
Could you please check whether this change affects `segment_matmul` performance?

Reply:
Verified, no impact on `segment_matmul`.